Description: The H5FD_class_t struct gained an extra member in 1.10
Bug-Debian: https://bugs.debian.org/846372
Author: Iain Lane <iain.lane@canonical.com>
--- a/ThirdParty/xdmf2/vtkxdmf2/libsrc/XdmfH5Driver.cxx
+++ b/ThirdParty/xdmf2/vtkxdmf2/libsrc/XdmfH5Driver.cxx
@@ -139,7 +139,11 @@
 #if (H5_VERS_MAJOR>1)||((H5_VERS_MAJOR==1)&&(H5_VERS_MINOR>=8))
 static haddr_t H5FD_dsm_get_eoa(const H5FD_t *_file, H5FD_mem_t type);
 static herr_t H5FD_dsm_set_eoa(H5FD_t *_file, H5FD_mem_t type, haddr_t addr);
+#if (H5_VERS_MAJOR>1)||((H5_VERS_MAJOR==1)&&(H5_VERS_MINOR>=10))
+static haddr_t H5FD_dsm_get_eof(const H5FD_t *_file, H5FD_mem_t type);
+#else
 static haddr_t H5FD_dsm_get_eof(const H5FD_t *_file);
+#endif
 #else
 static haddr_t H5FD_dsm_get_eoa(H5FD_t *_file);
 static herr_t H5FD_dsm_set_eoa(H5FD_t *_file, haddr_t addr);
@@ -155,6 +159,9 @@
     "dsm",                      /*name          */
     MAXADDR,                    /*maxaddr       */
     H5F_CLOSE_WEAK,             /*fc_degree     */
+#if (H5_VERS_MAJOR>1)||((H5_VERS_MAJOR==1)&&(H5_VERS_MINOR>=8))
+    NULL,                       /* terminate    */
+#endif
     NULL,                       /*sb_size       */
     NULL,                       /*sb_encode     */
     NULL,                       /*sb_decode     */
@@ -687,7 +694,9 @@
  *-------------------------------------------------------------------------
  */
 static haddr_t
-#if (H5_VERS_MAJOR>1)||((H5_VERS_MAJOR==1)&&(H5_VERS_MINOR>=8))
+#if (H5_VERS_MAJOR>1)||((H5_VERS_MAJOR==1)&&(H5_VERS_MINOR>=10))
+H5FD_dsm_get_eof(const H5FD_t *_file, H5FD_mem_t type)
+#elif (H5_VERS_MAJOR>1)||((H5_VERS_MAJOR==1)&&(H5_VERS_MINOR>=8))
 H5FD_dsm_get_eof(const H5FD_t *_file)
 #else
 H5FD_dsm_get_eof(H5FD_t *_file)
